### ISSUE TITLE
Create database using LiteDB.Studio?

### ISSUE THREAD

**Issue by bdaniel7** [mbdavid/LiteDB.Studio#70]
Is there a way to create a database using the Studio? Can you add this feature?

**Comment by Aaron-G-42** [comment]
Yes, go to Connect then for "Filename" specify a new (non-existent) file name.

For reference, in Sqlite Studio you open an empty file to get started... so this method of referencing non-existent was not intuitive to me.
